VSRAD.DebugServer/VSRAD.DebugServer.csproj

```diff
@@ -6,7 +6,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Packaging.Tools.Trimming" Version="1.1.0-preview1-26619-01" />
+    <PackageReference Include="Microsoft.Packaging.Tools.Trimming" Version="1.1.0-preview1-26619-01"/>
```

Comment:
I was able to build fully functional Server in both Debug and Release configurations without this reference. I suppose it appeared here by accident

Reply:
Not 100% sure about this one, caching and building in VS studio is not straightforward by any means, but it looks like this package is not necessary 👀

Reply:
Oh, okay, looks like it were been for ages. Sorry, than it is off-topic regarding this PR, maybe we should revisit all the deps of the project someday 👀👀